language/Pattern-Matching.md

```diff
@@ -23,16 +23,17 @@ Patterns can also be used when introducing functions. For example:
 example x y z = x * y + z
 ```
 
-The following pattern types are supported:
+The following forms can be used for matching:
```

Comment:
I like this change, as "form" has fewer connotations than "type", which could confuse newcomers.